base/db/R/query.traits.R

```diff
@@ -50,7 +50,7 @@ query.traits <- function(ids, priors, con = NULL,
   traits <- (dplyr::tbl(con, "traits")
     %>% dplyr::inner_join(dplyr::tbl(con, "variables"), by = c("variable_id" = "id"))
     %>% dplyr::filter(
-      (rlang::UQ(id_type) %in% ids),
+      (!!id_type %in% ids),
```

Comment:
UQ was an attempt to avoid check warnings, but I haven't checked on it in a while. If !! now works without a namespace qualifier, then 👍

Reply:
Here's what the `rlang` folks say as of [0.3.0](https://rlang.r-lib.org/news/index.html#deprecated-functions-and-arguments). I did this mostly to silence the annoying runtime warning about `rlang::UQ` being deprecated.

Reply:
My understanding of this is that `!!` and `!!!` aren't really functions so much as some kind of Domain Specific Programming magic operators. [Here](https://github.com/r-lib/rlang/blob/master/R/quasiquotation.R#L108-L134) is what the `rlang` source code has to say about them and why the namespaced forms are wrong.

As for how `R CMD check` will respond to this...🤷‍♂️

Reply:
I just ran `devtools::check` and didn't see any messages related to `!!`, so I think this is OK.

Reply:
<del>Still baffled how it works, but</del><ins>oh yeah, duh, we were [importing it already](https://github.com/PecanProject/pecan/blob/develop/base/db/R/zzz.R#L9)</ins>. Yep, this looks good and sorry for the extra noise.